Makefile

```diff
@@ -54,94 +59,16 @@ graph:
-
-webdevops/ansible:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/bootstrap
-
-webdevops/base:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/base
-
-webdevops/base-app:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/base-app
-
-webdevops/php:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/php
-
-webdevops/php-dev:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/php-dev
-
-webdevops/apache:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/apache
-
-webdevops/apache-dev:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/apache-dev
-
-webdevops/nginx:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/nginx
-
-webdevops/nginx-dev:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/nginx-dev
-
-webdevops/php-apache:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/php-apache
-
-webdevops/php-apache-dev:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/php-apache-dev
-
-webdevops/php-nginx:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/php-nginx
-
-webdevops/php-nginx-dev:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/php-nginx-dev
-
-webdevops/hhvm:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/hhvm
-
-webdevops/hhvm-apache:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/hhvm-apache
-
-webdevops/hhvm-nginx:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/hhvm-nginx
-
-webdevops/ssh:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/ssh
-
-webdevops/storage:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/storage
-
-webdevops/vsftp:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/vsftp
-
-webdevops/postfix:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/postfix
-
-webdevops/mail-sandbox:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/mail-sandbox
-
-webdevops/typo3:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/typo3
-
-webdevops/piwik:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/piwik
-
-webdevops/samson-deployment:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/samson-deployment
-
-webdevops/sphinx:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/sphinx
-
-webdevops/varnish:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/varnish
-
-webdevops/certbot:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/certbot
-
-webdevops/liquibase:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/liquibase
+	docker run -t -i --rm -p 8000 \
```

Comment:
Separated instruction over several lines to make easier to read and fit within 120 character line limit.